2023.1/index.adoc

```diff
@@ -0,0 +1,37 @@
+include::../shared/issue-footnotes.adoc[]
+
+:m3: ./metametamodel/metametamodel
+:refarch: ../reference-architecture/reference-architecture
+:serialization: ./serialization/serialization
+
+:emquad: &#x2001;
+
+= LionWeb Version 2023.1
+
+Stable <<{serialization}.adoc, serialization format>> (including JSON schema)
+and <<{m3}.adoc, meta-metamodel>> describing
+<<{m3}.adoc#Language, Languages>>,
+<<{m3}.adoc#Concept, Concepts>>,
+<<{m3}.adoc#Interface, Interfaces>>,
+<<{m3}.adoc#Annotation, Annotations>>,
+<<{m3}.adoc#Enumeration, Enumerations>>,
+<<{m3}.adoc#PrimitiveType, PrimitiveTypes>>,
+<<{m3}.adoc#Property, Properties>>,
+<<{m3}.adoc#Containment, Containments>>,
+and <<{m3}.adoc#Reference, References>>.
+
+The standard library contains classifiers <<{m3}.adoc#Node, Node>>
+and <<{m3}.adoc#INamed, INamed>>,
+and primitive types <<{m3}.adoc#Boolean, Boolean>>,
+<<{m3}.adoc#String, String>>,
+<<{m3}.adoc#Integer, Integer>>,
+and <<{m3}.adoc#JSON, JSON>>.
+
+== Contents
+
+* Serialization format{emquad}<<{serialization}.adoc, html>>{emquad}link:metametamodel/metametamodel.pdf[pdf]
+
+* Meta-metamodel{emquad}<<{m3}.adoc, html>>{emquad}link:serialization/serialization.pdf[pdf]
+
+* #TODO: Also reference architecture?#
```

Comment:
Should we include the reference architecture? It might change in the future, and we refer to it.

Reply:
Referring to it would be good, IMHO. I don't think we need to version the reference architecture (explicitly).

Reply:
Then we'd need to care for it in the case of future changes. And nobody will remember that.

Reply:
True: I guess it's best to remove it, then.